### PR TITLE
adjust readme

### PR DESCRIPTION
**Pull request by leonardoalt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,33 @@
...
-## Basic Concept
+For detailed documentation please visit [the powdr book](https://docs.powdr.org/).
 
 *powdr* is a toolkit that helps build zkVMs and similar proof frameworks.
 
 It has two main components:
 
-- A polynomial identity language that allows you to define polynomial constraints, lookups, etc.
-- An extensible assembly language to perform dynamic executions.
+- powdr-asm: an extensible assembly IR language to perform dynamic executions.
```
Comment (chriseth) [inline README.md:18]: would the word `self-defining` be fitting?

Comment (leonardoalt) [reply README.md:18]: where exactly?

If for the language itself I don't really like it. The language exists with very few builtins, but they exist: register datatypes, register assignments, machines, inline constraints, instructions, labels.

How about:

- powdr-asm: an extensible assembly IR language to define any VM / instruction set

?

Comment (chriseth) [reply README.md:18]: I'm mainly thinking about the fact that the language is not just `extensible`, but much more. But I'm also fine with it as is.

**Review by chriseth — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (leonardoalt) [reply README.md:18]: ok, this sentence is mostly the same, we can re-visit in the future

**Pull request merged**